examples/with-sentry/next.config.js

```diff
@@ -21,6 +21,7 @@ const COMMIT_SHA =
   VERCEL_BITBUCKET_COMMIT_SHA
 
 process.env.SENTRY_DSN = SENTRY_DSN
+const basePath = ''
```

Comment:
Hmm, isn't this just a no-op? Or what is the user supposed to update `basePath` to be?